### ISSUE TITLE
payment_stripe_sca for Odoo v10

### ISSUE THREAD

**Issue by wpichler** [odoo/odoo#36662]
Impacted versions:
v10

Steps to reproduce:
14. Sep. PSD2 Rules...

Current behavior:
Is not PSD2 compliant

Expected behavior:
Should be PSD2 compliant

Video/Screenshot link (optional):
https://www.odoo.com/blog/odoo-news-5/post/get-your-odoo-ready-for-strong-customer-authentication-sca-553

Customers are waiting for this on Odoo 10 - please tell us when it will be ready !

**Comment by pedrobaeza** [comment]
@dbo-odoo are there plans to backport it to v10?

**Comment by Feyensv** [comment]
Yes, V10 backport is in progress on https://github.com/odoo/odoo/pull/35688/ ;).

**Comment by pedrobaeza** [comment]
Thanks for the followup, then closing this.
